scripts/geome_things.py

```diff
@@ -147,6 +147,9 @@ def loadGEOMEEntries(session, max_count, start_from=None):
 @click.option(
     "-d", "--db_url", default=None, help="SQLAlchemy database URL for storage"
 )
+@click.option(
+    "-s", "--solr_url", default=None, help="Solr index URL"
+)
```

Comment:
You'll see these same changes in all the bootstrapping/boilerplate `things` scripts.  If we didn't have anything else to do, I think we could unify all the scripts in a much cleaner way.